app/src/test/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -30,52 +29,31 @@ internal class PopularMoviesViewModelTest {
     )
     val getPopularMoviesUseCase: GetPopularMoviesUseCase = mockk()
     val popularMoviesFragmentNavigator: PopularMoviesFragmentNavigator = mockk()
+    val successState = UIState.Success(movies)
+    val loadingState = UIState.Loading
+    val emptyState = UIState.Empty
 
     @Test
-    fun `when getPopularMoviesUseCase called and load data should set popularMovies`() = runTest {
+    fun `when getPopularMoviesUseCase called and load data, UIState should be in Success state`() {
         coEvery { getPopularMoviesUseCase() } returns moviesModel
         val tested = PopularMoviesViewModel(getPopularMoviesUseCase, popularMoviesFragmentNavigator)
```

Comment:
You can declare tested once, with by lazy, to follow the DRY rule, as junit tests are by default scoped as per test method, so every single test, in this case, will trigger as a fresh test

Reply:
Done 😄